### PR TITLE
comment nit: miners don't vote

### PR DESCRIPTION
**Pull request by instagibbs**

### CODE REVIEW

**Comment by gmaxwell** [comment]
ACK.

**Comment by MarcoFalke** [comment]
ACK e39dc69

**Comment by jonasschnelli** [comment]
ACK e39dc698adf46dc3306526971ca6f0f6401f30ca

File: src/main.cpp [hunk 1]
```diff
@@ -1886,8 +1886,8 @@ bool CheckInputs(const CTransaction& tx, CValidationState &state, const CCoinsVi
                     // such nodes as they are not following the protocol. That
                     // said during an upgrade careful thought should be taken
                     // as to the correct behavior - we may want to continue
-                    // peering with non-upgraded nodes even after a soft-fork
-                    // super-majority vote has passed.
+                    // peering with non-upgraded nodes even after soft-fork
+                    // super-majority signaling has occurred.
```
Comment (paveljanik) [inline src/main.cpp]: Sorry for being a nit-picker again, but isn't "signalling" the correct spelling?

**Comment by paveljanik** [comment]
ACK https://github.com/bitcoin/bitcoin/pull/8143/commits/e39dc698adf46dc3306526971ca6f0f6401f30ca

File: src/main.cpp (hunk 1 above)
Comment (instagibbs) [inline src/main.cpp]: it's 'Murika spelling, apologies

**Pull request merged**